solution/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.9.1-slim
```

Comment:
we suggest using `pytorch/pytorch:*-cuda*-cudnn*-runtime` images as base, as we found them to be hassle-free when working with AWS GPU instances

Reply:
changed and all prepared for first test on minimum workers